s3-proxy/index.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const morgan = require("morgan");
+const AWS = require("aws-sdk");
+
+const S3_BUCKET = process.env.S3_BUCKET;
+const S3_BUCKET_PREFIX = process.env.S3_BUCKET_PREFIX || "";
+const PORT = parseInt(process.env.PORT || "3000", 10);
```

Comment:
Please either add those to the Dockerfile as `ENV` commands or add a short README file for this image mentioning those, just for reference.